terraform/maintenance/live_service_url.tf

```diff
@@ -3,25 +3,124 @@ data "aws_route53_zone" "claim-power-of-attorney-refund_service_gov_uk" {
   name     = "claim-power-of-attorney-refund.service.gov.uk"
 }
 
+# public front cloudfront distribution
+locals {
+  public_front_cloudfront_distribution_domain_name = "d3pm2jfxjwrqjc.cloudfront.net"
+  public_front_cloudfront_distribution_zone_id     = "Z2FDTNDATAQYW2"
+}
+
+# public front load balancers
+data "aws_lb" "new_production_public_front" {
+  provider = aws.new_refunds_production
+  name     = "production-public-front"
+}
+
+resource "aws_route53_record" "claim-power-of-attorney-refund_service_gov_uk" {
+  provider = aws.old_refunds_production
+  zone_id  = "${data.aws_route53_zone.claim-power-of-attorney-refund_service_gov_uk.zone_id}"
+  name     = "claim-power-of-attorney-refund.service.gov.uk"
+  type     = "A"
+
+  alias {
+    evaluate_target_health = false
+    # point to old production cloudfront distribution
```

Comment:
OPTIONAL: Consider using a local var to switch this behaviour, rather than commenting in and out.